javalang/parser.py

```diff
@@ -1805,6 +1836,27 @@ def parse_expression_3(self):
         return primary
 
     @parse_debug
+    def parse_lambda_expression(self):
+        lambda_expr = None
+        parameters = None
+        with self.tokens:
+            if self.would_accept('('):
+                if self.tokens.look(i=2).value == ',':
+                    self.accept('(')
+                    parameters = []
+                    while not self.would_accept(')'):
```

Comment:
This doesn't look like it will be strict enough about requiring commas where they are necessary. Could you try adding a test for something like `(a b c) -> {};`?